Overview.src.html

```diff
@@ -1557,15 +1557,21 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
 <var>stream</var>, run these steps:
 
 <ol>
- <li><p>Call <code data-anolis-spec=streams>EnqueueInReadableStream</code>(<var>stream</var>,
+ <li><p>Let <var>controller</var> be <var>stream</var>.[[readableStreamController]].
+
+ <li><p>Call
+ <span data-anolis-spec=streams>ReadableStreamDefaultControllerEnqueue</span>(<var>controller</var>,
```

Comment:
If we only use it once, we might as well pass `stream.[[readableStreamController]]` in directly, no?